examples/SimpleRagas/assets/example_setupcode.py

```diff
@@ -0,0 +1,68 @@
+import asyncio
+import os
+import re
+import tempfile
+import lancedb
+
+import braintrust
+import markdownify
+import openai
+import requests
+
+# We're only going to embed a small subset of our total data for this example, but you can increase this number to embed more sections!
+NUM_SECTIONS = 20
+EMBEDDING_MODEL = "text-embedding-3-small"
+
+
+coda_qa_content_data = requests.get(
+    "https://gist.githubusercontent.com/wong-codaio/b8ea0e087f800971ca5ec9eef617273e/raw/39f8bd2ebdecee485021e20f2c1d40fd649a4c77/articles.json"
+).json()
+
+markdown_sections = [
+    {"doc_id": row["id"], "markdown": section.strip()}
+    for row in coda_qa_content_data
+    for section in re.split(r"(.*\n=+\n)", markdownify.markdownify(row["body"]))
+    if section.strip() and not re.match(r".*\n=+\n", section)
+]
+
+openai_client = braintrust.wrap_openai(
+    openai.AsyncOpenAI(
+        base_url="https://braintrustproxy.com/v1",
+        default_headers={"x-bt-use-cache": "always"},
+        api_key=os.environ.get("OPENAI_API_KEY", "Your OPENAI_API_KEY here"),
+    )
+)
+
+LANCE_DB_PATH = os.path.join(tempfile.TemporaryDirectory().name, "docs-lancedb")
+
+@braintrust.traced
```

Comment:
This braintrust code hurts a bit..we don't use anything from this in the cookbook so why have it? The embedding code could be a lot shorter w/o the braintrust logging

Reply:
This code is ancient :). If you just call `wrap_openai` on the braintrust client, it will automatically capture the embedding metrics.

Reply:
You don't need the `@braintrust.traced`? Neato

Reply:
You do to trace things generally. However in this case, as part of `wrap_openai`, we automatically instrument embedding calls. So just like you don't need to `@braintrust.traced` `openai.chat.completions.create`, you don't need to for embedding calls.